tools/gcp/comp_test/run_comp_test.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python2.7
```

Comment:
There are two points you raised.
a) the cmds sh script can be maintained externally to the docker image;
b) extra scripts introduced are unnecessary.

For a), I understand.  There are pros and cons of keeping the script within docker image.  The biggest benefit is that we potentially can run the comp test independently from github so we are not tied to any github repo version control.  In fact, if we go for an extra step (which I prefer but yet to introduce), we could run these tests without even a local github clone, simply by doing
  gcloud docker -- pull gcr.io/grpc-testing/java:v1.0.0; docker run ... bash -c ./run-all-saved_tests.sh
We are not quite there yet, but that is the direction I'd like to suggest.  Current PR is a step towards this direction.

Admittedly, the drawback of distributing the test scripts with the docker image is that it makes managing the test cases  across all releases a bit more work, especially when we try to globally adding/deleting tests.  However, I feel it is not necessarily a deal breaker.

For b), I respectively disagree.  At the bare minimum, you still need to
1.  run_interop_tests.py --manual_run
2.  rename the scripts generated and submit a PR in github
3.  upload the images to GCR and maintain a mapping of script in github <-> corresponding gcr images.
4. clean up the images.
5. For running the tests, download the gcr image.
6. change the image tag used in the shell script to reference the gcr image rather than the original image (containing uuid tag).
7. run the script and format the result to be consumed by kokoro etc.
8. clean up.

Step 1-4 need to be invoked every time when someone creates a snapshot.  Step5-8 need to invoked from kokoro unattended.  Hence, we do need scripts to standardize the steps instead of writing text based instructions for people (or kokoro) to follow. 

@makdharma @nicolasnoble Any comment here?